R/subsetting.R

```diff
@@ -0,0 +1,169 @@
+#' Subsetting tibbles
+#'
+#' @description
+#' Accessing columns, rows, or cells via `$`, `[[`, or `[` works very much like
+#' [with regular data frames][base::Extract.data.frame]. However, the
+#' behavior is different for tibbles and data frames in some cases:
+#' * `[` always returns a tibble by default, even if
```

Comment:
I don't think we should mention `drop` here at all.

Reply:
Moved to `@details`.